pkg/adaptor/cloud/libvirt/image.go

```diff
@@ -25,109 +21,131 @@ type image interface {
 	string() string
 }
 
-type httpImage struct {
-	url *url.URL
+// type httpImage struct {
```

Comment:
If this code is no longer being used, you can remove it the same way you removed imports.